rasa/core/policies/policy.py

```diff
@@ -290,6 +296,26 @@ def load(cls, path: Union[Text, Path]) -> "Policy":
                 featurizer = TrackerFeaturizer.load(path)
                 data["featurizer"] = featurizer
 
+            if "should_finetune" in kwargs:
+                data["should_finetune"] = kwargs["should_finetune"]
```

Comment:
shouldn't we do something like `data.update(kwargs)` to include all kwargs into `data`?

Reply:
Are we sure that every `kwarg` given to `load` should be forwarded to `__init__`? 
I just worried that we could end up passing stuff to the policy constructor which we didn't want to, and which ends up being automatically incorporated into the policy config (like in the TED policy).

Reply:
I'd argue that if somebody has a `load` function which uses `context` which isn't used by the `constructur` they should 1) `pop` it and 2) they would define the constructor call manually anyway. What do you think?

Reply:
Yeah that seems reasonable.